include/sparrow/array/data_type.hpp

```diff
@@ -443,37 +443,36 @@ namespace sparrow
     }
 
     /// @returns The number of bytes required to store the provided primitive data type.
-    constexpr std::size_t primitive_bytes_count(data_type data_type, int64_t length)
+    constexpr std::size_t primitive_bytes_count(data_type data_type, size_t length)
```

Comment:
Why this change?
My understanding is that arrow lengths and offsets are all stored in a 64bit integer, so this will break in 32bits?

Reply:
Fixed